README.md

```diff
@@ -37,17 +37,17 @@ The following options are available when configuring NuORDER:
 6. `oauth_token_secret`
 7. `oauth_callback` Default: `'oob'`
     * NuORDER will POST the callback the `oauth_verifier` token.
-    * If the callback is set to `'oob'`, the `oauth_verifier` token must be supplied manually by the user. 
+    * If the callback is set to `'oob'`, the `oauth_verifier` token must be supplied manually by the user.
 
 #### Example
 
 ```ruby
-Nuorder.configure do |config|
-  config.oauth_consumer_key = 'key'
-  config.oauth_consumer_secret = 'key'
-  config.oauth_token = 'key'
-  config.oauth_token_secret = 'key'
-end
+Nuorder.new(
```

Comment:
This example doesn't really make sense now since it's just instantiating a Client and not using it